### ISSUE TITLE
Frame Pre and Post processing for the model

### ISSUE THREAD

**Issue by sdp009** [MediaTek-NeuroPilot/mai22-real-time-video-sr#6]
Dear Team,

I was not able to find any normalization/pre/post-processing done on input images.

The script is loading ".png" file as it is, doing inference and saving output as is.

### Saving model prediction = https://github.com/MediaTek-NeuroPilot/mai22-real-time-video-sr/blob/main/generate_output.py#L71

Is this a expected by design or pre/post-processing is missing.

When I am trying model with sample video frames outside of validation set, it is generating checkboard artifacts.

**Comment by XUSean0118** [comment]
We do not apply normalization/pre/post-processing on input images except clipping output (https://github.com/MediaTek-NeuroPilot/mai22-real-time-video-sr/blob/main/model/mobile_rrn.py#L72).

The model is trained on image pair of HR and bicubic downsampling LR, so its generalizability may be constrained. A fine-tuned model should be trained if you want to apply it on other cases SR.

**Comment by sdp009** [comment]
Thanks @XUSean0118 for your inputs and confirmation !

Any suggestion for the attached checkerboard artifacts output which I am seeing, even when using REDS dataset (this is with all variants of the model : mobile_rnn_64/32/16 with using checkpoint for repo 'snapshot' directory)

![000_00000003](https://user-images.githubusercontent.com/32260424/209824888-d9467f2f-fcd4-420d-a7a1-08cbbab5eb77.png)

Does model need more training ?

**Comment by sdp009** [comment]
Command used for generating output is :

python3 generate_output.py --model_path model/mobile_rrn.py --model_name MobileRRN --ckpt_path snapshot/mobile_rrn_64/ckpt-150.index --data_dir data/REDS/train/train_sharp_bicubic/X4 --output_dir results

**Comment by XUSean0118** [comment]
It seems that the checkpoint is not correctly resorted.

The command should be :
python3 generate_output.py --model_path model/mobile_rrn.py --model_name MobileRRN **--ckpt_path snapshot/mobile_rrn_64/ckpt-150** --data_dir data/REDS/train/train_sharp_bicubic/X4 --output_dir results

**Comment by sdp009** [comment]
Thank you very much @XUSean0118 for your help. 
Indeed the checkpoint was not correctly resorted.
